lib/config/version.rb

```diff
@@ -1,3 +1,3 @@
 module Config
-  VERSION = '1.7.1'
+  VERSION = '1.7.2'.freeze
```

Comment:
You should not bump the version. `freeze` is OK

Reply:
I figured it needed to be bumped so bundler recognizes a new version when it's pushed. I'm used to doing it myself for work, forgot that might be part of a different process elsewhere.

Reply:
I will upgrade this whenever the gem will get released. Saying that, I can release once this PR is merged I guess.